### ISSUE TITLE
Fedora releases have mismatched versions

### ISSUE THREAD

**Issue by goodevilgenius** [jellyfin/jellyfin#5680]
**Describe the bug**

Packages provided at https://repo.jellyfin.org/releases/server/fedora/stable/ have mismatched versions.

**System (please complete the following information):**
 - OS: Fedora
 - Virtualization: none
 - Clients: Browser
 - Browser: n/a
 - Jellyfin Version: 10.7.1-1

**To Reproduce**
1. Go to https://repo.jellyfin.org/releases/server/fedora/stable/
2. Download three available packages:
    - jellyfin-10.7.1-1.fc33.x86_64.rpm
    - jellyfin-server-10.7.1-1.fc33.x86_64.rpm
    - jellyfin-web-10.7.1-1.fc31.noarch.rpm
3. Install all three with `rpm -i jellyfin*`, or `dnf install jellyfin*`

**Expected behavior**

Versions should match, and everything should install.

**Logs**

```
$ sudo dnf install jellyfin-*
Last metadata expiration check: 2:19:42 ago on Fri 02 Apr 2021 08:18:30 AM CDT.
Error: 
 Problem: conflicting requests
  - nothing provides jellyfin-web = 10.7.1-1.fc33 needed by jellyfin-10.7.1-1.fc33.x86_64
(try to add '--skip-broken' to skip uninstallable packages)
```

**Additional context**

The issue is that jellyfin-10.7.1-1.fc33 depends on jellyfin-server-10.7.1-1.fc33 and jellyfin-web-10.7.1-1.fc33. However, jellyfin-web-10.7.1-1.fc33 isn't provided. The version for 31 is provided instead.

It's the correct jellyfin version, but compiled for the wrong version of Fedora.

**Comment by goodevilgenius** [comment]
I was able to work around the issue.

Since `jellyfin` is just a meta package that requires the other two, but doesn't provide any actual files, it was safe to remove that one.

So, I first uninstalled my existing `jellyfin` package (which was currently at 10.6.4-1.fc31). Then I installed the `server` and `web` packages both. Everything seems to be working fine.

```
dnf uninstall jellyfin
dnf install jellyfin-server-10.7.1-1.fc33.x86_64.rpm jellyfin-web-10.7.1-1.fc31.noarch.rpm
```
